.gitignore

```diff
@@ -16,78 +16,79 @@ bin/
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 # Copied from https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Global/JetBrains.gitignore
 
- # User-specific stuff
```

Comment:
Any idea what changed here?

Reply:
Removed one white space at the beginning for all the lines